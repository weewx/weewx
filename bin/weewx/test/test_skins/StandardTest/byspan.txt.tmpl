Half Hour Averages in Last Hour
#for $_span in $last_hour.spans(interval=1800)
$_span.dateTime.format("%a %H:%M"): $_span.outTemp.avg
#end for

Hourly Top Temperatures in Last Day
#for $_hour in $last_day.hours
$_hour.dateTime.format("%a %m/%d %H:%M"): $_hour.outTemp.max at $_hour.outTemp.maxtime
#end for

Daily Top Temperatures in Last Week
#for $_day in $last_week.days
$_day.dateTime.format("%a %m/%d"): $_day.outTemp.max at $_day.outTemp.maxtime
#end for

Daily Top Temperatures in Last Month
#for $_day in $last_month.days
$_day.dateTime.format("%a %m/%d"): $_day.outTemp.max at $_day.outTemp.maxtime
#end for

Monthly Top Temperatures in Last Year
#for $_month in $last_year.months
$_month.dateTime.format("%b %Y"): $_month.outTemp.max at $_month.outTemp.maxtime
#end for

3 hour averages over last day
DateTime,outTemp
#for $span in $last_day.spans(interval=10800)
$span.dateTime.format("%m/%d %H:%M:%S"),$span.outTemp.avg
#end for

All Temp records in last day
#for $record in $last_day.records
$record.dateTime.format("%m/%d %H:%M:%S"),$record.outTemp
#end for