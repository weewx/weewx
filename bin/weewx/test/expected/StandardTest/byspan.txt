Half Hour Averages in Last Hour
Fri 10:00: 45.9F
Fri 10:30: 48.0F
Fri 11:00: 49.9F

Hourly Top Temperatures in Last Day
Thu 09/02 11:00: 54.1F at 12:00
Thu 09/02 12:00: 59.3F at 13:00
Thu 09/02 13:00: 64.5F at 14:00
Thu 09/02 14:00: 69.3F at 15:00
Thu 09/02 15:00: 73.4F at 16:00
Thu 09/02 16:00: 76.5F at 17:00
Thu 09/02 17:00: 78.5F at 18:00
Thu 09/02 18:00: 79.1F at 19:00
Thu 09/02 19:00: 79.1F at 19:10
Thu 09/02 20:00: 78.2F at 20:10
Thu 09/02 21:00: 76.0F at 21:10
Thu 09/02 22:00: 72.6F at 22:10
Thu 09/02 23:00: 68.3F at 23:10
Fri 09/03 00:00: 63.3F at 00:10
Fri 09/03 01:00: 58.1F at 01:10
Fri 09/03 02:00: 53.0F at 02:10
Fri 09/03 03:00: 48.2F at 03:10
Fri 09/03 04:00: 44.2F at 04:10
Fri 09/03 05:00: 41.2F at 05:10
Fri 09/03 06:00: 39.3F at 06:10
Fri 09/03 07:00: 39.5F at 08:00
Fri 09/03 08:00: 41.5F at 09:00
Fri 09/03 09:00: 44.6F at 10:00
Fri 09/03 10:00: 48.7F at 11:00
Fri 09/03 11:00: 50.3F at 11:20

Daily Top Temperatures in Last Week
Fri 08/27: 82.7F at 19:00
Sat 08/28: 82.1F at 19:00
Sun 08/29: 81.5F at 19:00
Mon 08/30: 80.9F at 19:00
Tue 08/31: 80.3F at 19:00
Wed 09/01: 79.8F at 19:00
Thu 09/02: 79.1F at 19:00
Fri 09/03: 63.3F at 00:10

Daily Top Temperatures in Last Month
Wed 08/04: 93.6F at 19:00
Thu 08/05: 93.2F at 19:00
Fri 08/06: 92.9F at 19:00
Sat 08/07: 92.5F at 19:00
Sun 08/08: 92.1F at 19:00
Mon 08/09: 91.6F at 19:00
Tue 08/10: 91.2F at 19:00
Wed 08/11: 90.8F at 19:00
Thu 08/12: 90.3F at 19:00
Fri 08/13: 89.9F at 19:00
Sat 08/14: 89.4F at 19:00
Sun 08/15: 88.9F at 19:00
Mon 08/16: 88.5F at 19:00
Tue 08/17: 88.0F at 19:00
Wed 08/18: 87.5F at 19:00
Thu 08/19: 87.0F at 19:00
Fri 08/20: 86.5F at 19:00
Sat 08/21: 85.9F at 19:00
Sun 08/22: 85.4F at 19:00
Mon 08/23: 84.9F at 19:00
Tue 08/24: 84.3F at 19:00
Wed 08/25: 83.8F at 19:00
Thu 08/26: 83.2F at 19:00
Fri 08/27: 82.7F at 19:00
Sat 08/28: 82.1F at 19:00
Sun 08/29: 81.5F at 19:00
Mon 08/30: 80.9F at 19:00
Tue 08/31: 80.3F at 19:00
Wed 09/01: 79.8F at 19:00
Thu 09/02: 79.1F at 19:00
Fri 09/03: 63.3F at 00:10

Monthly Top Temperatures in Last Year
Sep 2009:    N/A at    N/A
Oct 2009:    N/A at    N/A
Nov 2009:    N/A at    N/A
Dec 2009: 0.0F at 01-Jan-2010 00:00
Jan 2010: 25.5F at 31-Jan-2010 18:00
Feb 2010: 38.8F at 28-Feb-2010 18:00
Mar 2010: 59.0F at 31-Mar-2010 19:00
Apr 2010: 78.8F at 30-Apr-2010 19:00
May 2010: 94.2F at 31-May-2010 19:00
Jun 2010: 100.0F at 30-Jun-2010 19:00
Jul 2010: 100.0F at 02-Jul-2010 19:00
Aug 2010: 94.7F at 01-Aug-2010 19:00
Sep 2010: 79.8F at 01-Sep-2010 19:00

3 hour averages over last day
DateTime,outTemp
09/02 09:00:00,48.1F
09/02 12:00:00,62.3F
09/02 15:00:00,74.9F
09/02 18:00:00,78.5F
09/02 21:00:00,70.6F
09/03 00:00:00,56.0F
09/03 03:00:00,43.2F
09/03 06:00:00,39.5F
09/03 09:00:00,45.8F

All Temp records in last day
09/02 11:30:00,51.7F
09/02 11:40:00,52.5F
09/02 11:50:00,53.3F
09/02 12:00:00,54.1F
09/02 12:10:00,55.0F
09/02 12:20:00,55.8F
09/02 12:30:00,56.7F
09/02 12:40:00,57.6F
09/02 12:50:00,58.4F
09/02 13:00:00,59.3F
09/02 13:10:00,60.2F
09/02 13:20:00,61.0F
09/02 13:30:00,61.9F
09/02 13:40:00,62.8F
09/02 13:50:00,63.6F
09/02 14:00:00,64.5F
09/02 14:10:00,65.3F
09/02 14:20:00,66.1F
09/02 14:30:00,66.9F
09/02 14:40:00,67.7F
09/02 14:50:00,68.5F
09/02 15:00:00,69.3F
09/02 15:10:00,70.0F
09/02 15:20:00,70.7F
09/02 15:30:00,71.4F
09/02 15:40:00,72.1F
09/02 15:50:00,72.7F
09/02 16:00:00,73.4F
09/02 16:10:00,74.0F
09/02 16:20:00,74.5F
09/02 16:30:00,75.1F
09/02 16:40:00,75.6F
09/02 16:50:00,76.1F
09/02 17:00:00,76.5F
09/02 17:10:00,76.9F
09/02 17:20:00,77.3F
09/02 17:30:00,77.7F
09/02 17:40:00,78.0F
09/02 17:50:00,78.3F
09/02 18:00:00,78.5F
09/02 18:10:00,78.7F
09/02 18:20:00,78.9F
09/02 18:30:00,79.0F
09/02 18:40:00,79.1F
09/02 18:50:00,79.1F
09/02 19:00:00,79.1F
09/02 19:10:00,79.1F
09/02 19:20:00,79.1F
09/02 19:30:00,79.0F
09/02 19:40:00,78.8F
09/02 19:50:00,78.7F
09/02 20:00:00,78.4F
09/02 20:10:00,78.2F
09/02 20:20:00,77.9F
09/02 20:30:00,77.6F
09/02 20:40:00,77.2F
09/02 20:50:00,76.8F
09/02 21:00:00,   N/A
09/02 21:10:00,76.0F
09/02 21:20:00,75.5F
09/02 21:30:00,75.0F
09/02 21:40:00,74.4F
09/02 21:50:00,73.8F
09/02 22:00:00,73.2F
09/02 22:10:00,72.6F
09/02 22:20:00,71.9F
09/02 22:30:00,71.2F
09/02 22:40:00,70.5F
09/02 22:50:00,69.8F
09/02 23:00:00,69.0F
09/02 23:10:00,68.3F
09/02 23:20:00,67.5F
09/02 23:30:00,66.7F
09/02 23:40:00,65.9F
09/02 23:50:00,65.0F
09/03 00:00:00,64.2F
09/03 00:10:00,63.3F
09/03 00:20:00,62.5F
09/03 00:30:00,61.6F
09/03 00:40:00,60.7F
09/03 00:50:00,59.9F
09/03 01:00:00,59.0F
09/03 01:10:00,58.1F
09/03 01:20:00,57.2F
09/03 01:30:00,56.4F
09/03 01:40:00,55.5F
09/03 01:50:00,54.6F
09/03 02:00:00,53.8F
09/03 02:10:00,53.0F
09/03 02:20:00,52.1F
09/03 02:30:00,51.3F
09/03 02:40:00,50.5F
09/03 02:50:00,49.7F
09/03 03:00:00,48.9F
09/03 03:10:00,48.2F
09/03 03:20:00,47.5F
09/03 03:30:00,46.8F
09/03 03:40:00,46.1F
09/03 03:50:00,45.4F
09/03 04:00:00,44.8F
09/03 04:10:00,44.2F
09/03 04:20:00,43.6F
09/03 04:30:00,43.0F
09/03 04:40:00,42.5F
09/03 04:50:00,42.0F
09/03 05:00:00,41.6F
09/03 05:10:00,41.2F
09/03 05:20:00,40.8F
09/03 05:30:00,40.4F
09/03 05:40:00,40.1F
09/03 05:50:00,39.8F
09/03 06:00:00,39.6F
09/03 06:10:00,39.3F
09/03 06:20:00,39.2F
09/03 06:30:00,39.0F
09/03 06:40:00,38.9F
09/03 06:50:00,38.9F
09/03 07:00:00,38.8F
09/03 07:10:00,38.9F
09/03 07:20:00,38.9F
09/03 07:30:00,39.0F
09/03 07:40:00,39.1F
09/03 07:50:00,39.3F
09/03 08:00:00,39.5F
09/03 08:10:00,39.7F
09/03 08:20:00,40.0F
09/03 08:30:00,40.3F
09/03 08:40:00,40.7F
09/03 08:50:00,   N/A
09/03 09:00:00,41.5F
09/03 09:10:00,41.9F
09/03 09:20:00,42.4F
09/03 09:30:00,42.9F
09/03 09:40:00,43.5F
09/03 09:50:00,44.0F
09/03 10:00:00,44.6F
09/03 10:10:00,45.3F
09/03 10:20:00,45.9F
09/03 10:30:00,46.6F
09/03 10:40:00,47.3F
09/03 10:50:00,48.0F
09/03 11:00:00,48.7F
09/03 11:10:00,49.5F
09/03 11:20:00,50.3F

